app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.kt

```diff
@@ -1,16 +1,14 @@
 package de.tum.`in`.tumcampusapp.component.tumui.calendar
 
 import android.os.Bundle
-
 import de.tum.`in`.tumcampusapp.R
-import de.tum.`in`.tumcampusapp.component.other.generic.activity.ActivityForAccessingTumOnline
-import de.tum.`in`.tumcampusapp.component.tumui.calendar.model.EventsResponse
+import de.tum.`in`.tumcampusapp.component.other.generic.activity.BaseActivity
 import de.tum.`in`.tumcampusapp.utils.Const
 
 /**
  * Activity showing the user's calendar. Calendar items (events) are fetched from TUMOnline and displayed as blocks on a timeline.
  */
-class CalendarActivity : ActivityForAccessingTumOnline<EventsResponse>(R.layout.activity_calendar) {
+class CalendarActivity : BaseActivity(R.layout.activity_calendar) {
```

Comment:
What's the reason for this change?

Reply:
Initially I thought only the CalendarActivity was having issues with the toolbar and that was the difference to the other activities, which were already moved to this class from the **deprecated ActivityForAccessingTumOnline**. This is also why I left it in, because the original Parent Class is marked as deprecated.

Reply:
![deprecated_activityforaccessingtumonline](https://user-images.githubusercontent.com/9247500/144236628-baf97c75-658d-4d73-b06f-53d03827425c.png)

Reply:
Alright I see, thanks :)